src/pages/guidelines/spacing.mdx

```diff
@@ -11,30 +11,26 @@ Carbon for IBM.com uses uniform components, elements, and spacing to encourage c
 
 <AnchorLinks>
 
-<AnchorLink>Designing with space</AnchorLink>
-<AnchorLink>Spacing token and scale</AnchorLink>
-<AnchorLink>Component spacing patterns</AnchorLink>
+<AnchorLink>Overview</AnchorLink>
+<AnchorLink>Spacing tokens and scale</AnchorLink>
+<AnchorLink>Spacing concepts</AnchorLink>
+<AnchorLink>Constructing a page with content components</AnchorLink>
 <AnchorLink>FAQ</AnchorLink>
-<AnchorLink>Support</AnchorLink>
+<AnchorLink>Feedback</AnchorLink>
 
 </AnchorLinks>
 
-## Designing with space
+## Overview
 
-Every part of a UI should be intentional, including the empty space, also known as white space, between elements. The amount of space between items creates relationships and hierarchy. You can learn more about white space and how space can help you create relationships and hierarchy on the [Carbon spacing guidelines](https://www.carbondesignsystem.com/guidelines/spacing/overview/#designing-with-space) page.
+Every part of a UI should be intentional, including the empty space, also known as white space, between elements. The amount of space between items creates relationships and hierarchy. You can learn more about white space and how space can help you create relationships and hierarchy with the [Carbon spacing guidelines](https://www.carbondesignsystem.com/guidelines/spacing/overview/#designing-with-space).
 
 When designing for IBM.com, the spacing scale and type tokens are used to ensure the information is well organized and consistent. See the [Type pairings guidance](../expressive-styling/type-pairing) to learn more about the type hierarchy of the Carbon for IBM.com components.
```

Comment:
```suggestion
When designing for IBM.com, the spacing scale and type tokens are used to ensure the information is well organized and consistent. See the [Type pairing guidance](../../expressive-styling/type-pairing) to learn more about the type hierarchy of the Carbon for IBM.com components.
```
this is giving a 404, I think we might need another ../ because it's within a tab?